subquery.py

```diff
@@ -99,7 +100,17 @@ def query_multiple_source(
     if telegram:
         raise NotImplementedError
     if github:
-        raise NotImplementedError
+        github_query_engine = GitHubQueryEngine(community_id=community_id).prepare()
+        tool_metadata = ToolMetadata(
+            name="GitHub",
+            description="Hosts code repositories and project materials from the GitHub platform.",
```

Comment:
Still the old description

Reply:
It might be nice to save all descriptions and other prompts in a single document. Then import them from there whenever we need them like an env var file. That could make updating prompts more easy in the future

Reply:
Thanks, that was because I started it from the previous branch yesterday. will consider updating it.

Reply:
It will be resolved when merging with main.